safekit-cluster-mirror/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"OS": {
+			"type": "string",
+			"metadata": {
+				"description": "Operating System to install"
+			},
+			"defaultValue": "2016-Datacenter",
+			"allowedValues": [
+				"2016-Datacenter",
+				"Linux CentOS"
+			]
+		},
+		"vmSize": {
+			"type": "string",
+			"metadata": {
+				"description": "the VM size for all nodes"
+			},
+			"defaultValue": "Standard_A2_v2"
+		},
+		"adminUser": {
+			"type": "string",
+			"metadata": {
+				"description": "User for the Virtual Machines."
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "Password for the Virtual Machines."
+			}
+		},
+		"VIPDnsLabel": {
+			"type": "string",
+			"metadata": {
+				"description": "Public VIP dns label."
+			},
+			"defaultValue": "[concat(resourceGroup().name,'vip')]"
+		},
+		"azurePwsh": {
+			"type": "string",
+			"metadata": {
+				"description": "install azure powershell module (optional)"
+			},
+			"defaultValue": "no",
+			"allowedValues": [
+				"yes",
+				"no"
+			]
+		},
+		"location": {
+			"type": "string",
+			"metadata": {
+				"description": "resources location"
+			},
+			"defaultValue": "westus2",
+			"allowedValues": [
```

Comment:
The workload uses availability zones, which are, if I am not mistaken, only available at the moment in the regions listed in the AllowedValues field. Setting resourceGroup().location is not an option since Travis validation would probably fail, and using a param seems too restrictive...   Since the 101-vm-simple-zones template was using the allowedValues/defaultValue trick to limit deployment to supported regions, I thought it was the way to go... Please clarify.

Reply:
Ok, then let's set the allowedValues to the locations that support zones: https://docs.microsoft.com/en-us/azure/availability-zones/az-overview#regions-that-support-availability-zones